public/js/components/Utilities/Modal.spec.js

```diff
@@ -5,15 +5,15 @@ import {shallow} from 'enzyme';
 
 test('Should render null when closed', () => {
   const component = renderer.create(
-    <Modal isOpen={false}><div>Modal Contents</div></Modal>
+    <Modal isOpen={false} dismiss={() => {}}><div>Modal Contents</div></Modal>
```

Comment:
So further down that's what we do, when we're actually testing if those functions are being called... I quite like that it looks empty when we're not testing anything to do with it.